tests/filecheck/dialects/snitch/snitch_ops.mlir

```diff
@@ -0,0 +1,32 @@
+// RUN: xdsl-opt %s | xdsl-opt | filecheck %s
+"builtin.module"() ({
+  %A = "test.op"() : () -> !riscv.reg<> // vector A: base address
+  %B = "test.op"() : () -> !riscv.reg<> // vector B: base address
```

Comment:
Just pointing it out: You could use one test op with two results.

Reply:
Thanks, TIL!